specification/baremetalinfrastructure/resource-manager/Microsoft.BareMetalInfrastructure/preview/2023-08-04-preview/baremetalinfrastructure.json

```diff
@@ -922,6 +922,12 @@
           "type": "integer",
           "format": "int32",
           "description": "Specifies the size of an empty data disk in gigabytes."
+        },
+        "lun": {
```

Comment:
I'm assuming I should also add lun back to the ALI spec?

Reply:
Please do not add the lun property in ALI. This object isn't present in our domain model. I've retained it because its removal would result in a breaking change. However, in ALI, removing the 'lun' doesn't constitute a breaking change. We can address the 'lun' issue in a subsequent iteration for BMI.